ctypes_override/ctypes_init.diff

```diff
@@ -0,0 +1,20 @@
+406a407,417
+> 
+> _ARCH_TRIPLET = {
+>     'arm64': 'aarch64-linux-gnu',
+>     'i386': 'i386-linux-gnu',
+>     'ppc64el': 'powerpc64le-linux-gnu',
+>     'powerpc': 'powerpc-linux-gnu',
+>     'amd64': 'x86_64-linux-gnu',
+>     's390x': 's390x-linux-gnu',
+> }
+> 
+> 
+421a433,439
+>         if name == 'libsodium.so' and _os.getenv('SNAP_NAME', '') == 'snapcraft':
```

Comment:
Why limit this to libsodium? Won't any lib that hits this code path suffer from the same issues?